packages/mdc-list/README.md

```diff
@@ -393,7 +395,7 @@ The MDCList JavaScript component implements the WAI-ARIA best practices for
 [Listbox](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox). This includes overriding the default tab behavior
 within the list component.
 
-The `tabindex` should be set to `0` for first list item element or selected list item element, remaining list item elements should not have `tabindex` set.
+The first item in the list should always have `tabindex` set to `0` on render, remaining list item elements should not have `tabindex` set so that entire list will have only one tab stop.
```

Comment:
Why did we change this advice (and the examples above) away from setting the tabindex on the selected item? Does the code rely on the first item having tabindex now?

Reply:
Reverted this. This is not mandatory but user can still prefer to set tabindex to first element whether it is selectable list or not.

Reply:
Should we also revert the changed examples then? If someone sets tabindex="0" on the first item but there is a selected item elsewhere in the list, will the selected item not have focus? (It should...)

Reply:
I think I just found the answer to this question when testing the test pages, which also have this issue. Setting tabindex="0" on the first item when something else is selected will indeed send focus to the first item when the list is first tabbed into... then when you tab out and in again, it will properly focus the first selected item. So the examples and test pages should definitely initialize tabindex on the first selected element for the best UX.

Reply:
I agree. The `tabindex=0` should be set to selected item or first item (if list is not selectable). All examples already have tabindex=0 set to appropriate item. Are there any other examples that I left out having incorrect tabindex?

Reply:
There are 3 examples in the readme which you changed from having tabindex on the selected item to having tabindex on the first item. When you changed the docs under these examples back, you didn't also change the examples back themselves.

The checkbox, radio, and single-select screenshot test pages all have tabindex on the first item rather than on the selected item.

Reply:
Done.